sdk/package.json

```diff
@@ -30,6 +30,7 @@
     "@types/js-base64": "^3.0.0",
     "@types/node": "^14.6.2",
     "assert-no-diff": "^3.0.4",
+    "assemblyscript-temporal": "^1.5.0",
```

Comment:
This Isn't needed since it's a dep of `near-sdk-core`.

Reply:
Well, we need `PlainDateTIme` object for tests,
https://github.com/ashutoshvarma/near-sdk-as/blob/3070ae69852e5376236ae1bc7cef6f74a3bc1ac8/sdk/assembly/__tests__/datetime.spec.ts#L2

So we might need it.

Reply:
Well since temporal is a transitive dependency, e.i. `near-sdk-as` --> `near-sdk-core` --> `temporal`, it already should be available.

Reply:
Ohh, my bad

Reply:
Done!